browser/net/brave_common_static_redirect_network_delegate_helper.cc

```diff
@@ -33,11 +33,26 @@ bool IsUpdaterURL(const GURL& gurl) {
       [&gurl](URLPattern pattern) { return pattern.MatchesURL(gurl); });
 }
 
+bool IsPDFJsExtensionURL(const GURL& gurl) {
+  // We are using pdfium as a pdf viewer. So, we make user can use pdf.js as a
+  // primary pdf document viewer.
+  // To do that, pdfjs install request should bypass brave component server.
```

Comment:
Need security review for this.

Reply:
We can back this patch out after verifying patch by @jumde 😄 
https://github.com/brave/go-update/pull/27

Reply:
Patch removed!